src/fhirdefs/load.ts

```diff
@@ -25,6 +25,12 @@ export async function loadDependency(
   FHIRDefs: FHIRDefinitions,
   cachePath: string = path.join(os.homedir(), '.fhir', 'packages')
 ): Promise<FHIRDefinitions> {
+  if (/[A-Z]/.test(packageName)) {
+    logger.warn(
+      `${packageName} contains uppercase characters, which is discouraged. SUSHI will use ${packageName.toLowerCase()} as the package name.`
+    );
+    packageName = packageName.toLowerCase();
+  }
```

Comment:
When I try to use a package with capitals, like:
```yaml
dependencies:
  hl7.fhir.us.Core: current
```
I'm seeing an issue down the line from SUSHI:
![image](https://user-images.githubusercontent.com/52747882/131885675-ea62e682-fb11-45d0-8cc2-78849e1bf34a.png)

This seems to happen because although we lowercase the package name here, we don't do that in the configuration, and that is the value that gets used in the publisher. I'm wondering if it may be easier to just move this earlier in the chain, and do this check as a part of importing the configuration itself. What do you think?

Reply:
It definitely makes more sense to handle this as the configuration file is being imported and parsed, rather than when each individual dependency is being loaded. Commit `327b977` moved the warning and lowercase-ification to when the configuration is imported.